src/components/Upload.vue

```diff
@@ -1,17 +1,19 @@
 <template lang="pug">
-.upload-wrapper
-  slot(name="toolbar")
-    v-toolbar(dense, dark, color="success")
-      v-toolbar-title Upload
-      v-spacer
-      v-btn(v-if="closeable", icon, flat, @click="$emit('close')")
-        v-icon close
-  .px-0.py-2
-    slot(name="header")
-      .title.px-3
-        v-avatar.mr-2(color="blue-grey lighten-4")
-          v-icon {{ ResourceIcons[dest._modelType.toUpperCase()] }}
-        span {{ dest.name }}
+v-card.upload-wrapper(flat)
+  v-card-title(primary-title)
+    div
+      slot(name="toolbar")
```

Comment:
I think this slot should go *outside* the `v-card-title` to make it more powerful. I can imagine downstreams wanting to remove the top part altogether to add their own title, toolbar, whatever, e.g. to make an upload dialog where only the list part scrolls while the title remains fixed.

Reply:
And we should rename the slot to "header" or something like that that corresponds to its semantics a bit more closely.

Reply:
Done, and renamed slot to "header".